### PR TITLE
Remove nullrender

### LINKED ISSUE #21502: Remove nullRenderer
Remove `nullRender`, replace it with conditional rendering in `renderComponent` functions to avoid unnecessary nodes in React tree.

### PR DESCRIPTION
**Pull request by bsunderhus**
## Current Behavior

`getSlots()` returns `nullRender` for "empty slots" which should render nothing.

## New Behavior

`getSlots()` returns `null`, it is `renderComponent()` responsibility to conditionally render optional slots.

## Related RFC

https://github.com/microsoft/fluentui/pull/21503

Fixes #21502

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                       |    Baseline (minified/GZIP) |                          PR |                                                                                                                                                                                                                                         Change |
| :-------------------------------------------------------------------------------------- | --------------------------: | --------------------------: | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr> | `21.474 kB`<br />`7.118 kB` | `21.499 kB`<br />`7.114 kB` | `25 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`-4 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /> |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr> |  `8.323 kB`<br />`3.479 kB` |  `8.321 kB`<br />`3.476 kB` |       `-2 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /><br />`-3 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /> |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                       |        Size (minified/GZIP) |
| ------------------------------------------------------------------------------------------------------- | --------------------------: |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePopper</abbr>   | `22.808 kB`<br />`7.935 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr> |        `189 B`<br />`161 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/e9a94d01c1a1f8214cc5dedc86bfa28a3ad8797d'>e9a94d01c1a1f8214cc5dedc86bfa28a3ad8797d</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-utilities/src/compose/getSlots.ts [hunk 1]
```diff
@@ -68,9 +67,9 @@ export function getSlots<R extends SlotPropsRecord>(
 function getSlot<R extends SlotPropsRecord, K extends keyof R>(
   state: ComponentState<R>,
   slotName: K,
-): readonly [React.ElementType<R[K]>, R[K]] {
+): readonly [React.ElementType<R[K]> | null, R[K]] {
```
Comment (miroslavstastny) [inline packages/react-utilities/src/compose/getSlots.ts:70]: Shouldn't this return `|null` only for nullable slots?

Comment (bsunderhus) [reply packages/react-utilities/src/compose/getSlots.ts:70]: `getSlot` function is only internally used by `getSlots`, it doesn't have enough types information to distinguish, the `null` only for nullable slots is treated by `getSlots` method

File: packages/react-card/src/components/CardHeader/renderCardHeader.tsx [hunk 2]
```diff
@@ -11,11 +11,13 @@ export const renderCardHeader_unstable = (state: CardHeaderState) => {
...
     <slots.root {...slotProps.root}>
       <slots.image {...slotProps.image} />
-      <slots.content {...slotProps.content}>
-        <slots.header {...slotProps.header} />
-        <slots.description {...slotProps.description} />
-      </slots.content>
-      <slots.action {...slotProps.action} />
+      {slots.content && (
+        <slots.content {...slotProps.content}>
+          <slots.header {...slotProps.header} />
```
Comment (miroslavstastny) [inline packages/react-card/src/components/CardHeader/renderCardHeader.tsx:16]: Not related to this PR, but there are required slots rendered inside an optional slot. Is that expected?
cc @khmakoto

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ContextualMenu-mount_base.html">16388</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ContextualMenu-mount.html">8492</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Avatar-mount_base.html">939</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Avatar-mount.html">823</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/BaseButton-mount_base.html">934</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/BaseButton-mount.html">956</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Breadcrumb-mount_base.html">2678</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Breadcrumb-mount.html">2693</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ButtonNext-mount_base.html">518</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ButtonNext-mount.html">495</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Checkbox-mount_base.html">1570</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Checkbox-mount.html">1574</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/CheckboxBase-mount_base.html">1336</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/CheckboxBase-mount.html">1334</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ChoiceGroup-mount_base.html">4858</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ChoiceGroup-mount.html">4872</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ComboBox-mount_base.html">1032</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ComboBox-mount.html">1036</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/CommandBar-mount_base.html">10355</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/CommandBar-mount.html">10369</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ContextualMenu-mount_base.html">16388</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ContextualMenu-mount.html">8492</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/DefaultButton-mount_base.html">1149</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/DefaultButton-mount.html">1189</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/DetailsRow-mount_base.html">3854</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/DetailsRow-mount.html">3814</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/DetailsRowFast-mount_base.html">3875</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/DetailsRowFast-mount.html">3785</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/DetailsRowNoStyles-mount_base.html">3684</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/DetailsRowNoStyles-mount.html">3639</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Dialog-mount_base.html">2674</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Dialog-mount.html">2690</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/DocumentCardTitle-mount_base.html">210</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/DocumentCardTitle-mount.html">182</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Dropdown-mount_base.html">3335</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Dropdown-mount.html">3305</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FluentProviderNext-mount_base.html">1941</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FluentProviderNext-mount.html">1928</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FluentProviderWithTheme-mount_base.html">170</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FluentProviderWithTheme-mount.html">162</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">110</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FluentProviderWithTheme-virtual-rerender.html">119</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">196</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">202</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FocusTrapZone-mount_base.html">1846</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FocusTrapZone-mount.html">1887</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FocusZone-mount_base.html">1899</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/FocusZone-mount.html">1903</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/IconButton-mount_base.html">1739</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/IconButton-mount.html">1764</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Label-mount_base.html">371</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Label-mount.html">379</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Layer-mount_base.html">3066</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Layer-mount.html">3093</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Link-mount_base.html">491</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Link-mount.html">511</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/MakeStyles-mount_base.html">1762</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/MakeStyles-mount.html">1729</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/MenuButton-mount_base.html">1520</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/MenuButton-mount.html">1550</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/MessageBar-mount_base.html">2034</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/MessageBar-mount.html">2024</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Nav-mount_base.html">3380</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Nav-mount.html">3310</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/OverflowSet-mount_base.html">1146</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/OverflowSet-mount.html">1200</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Panel-mount_base.html">2578</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Panel-mount.html">2594</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Persona-mount_base.html">869</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Persona-mount.html">880</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Pivot-mount_base.html">1483</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Pivot-mount.html">1474</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/PrimaryButton-mount_base.html">1323</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/PrimaryButton-mount.html">1342</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Rating-mount_base.html">7897</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Rating-mount.html">7747</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/SearchBox-mount_base.html">1379</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/SearchBox-mount.html">1374</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Shimmer-mount_base.html">2603</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Shimmer-mount.html">2635</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Slider-mount_base.html">2003</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Slider-mount.html">2022</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/SpinButton-mount_base.html">5071</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/SpinButton-mount.html">5166</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Spinner-mount_base.html">467</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Spinner-mount.html">454</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/SplitButton-mount_base.html">3243</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/SplitButton-mount.html">3220</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Stack-mount_base.html">556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Stack-mount.html">553</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/StackWithIntrinsicChildren-mount_base.html">2344</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/StackWithIntrinsicChildren-mount.html">2315</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/StackWithTextChildren-mount_base.html">5307</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/StackWithTextChildren-mount.html">5320</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/SwatchColorPicker-mount_base.html">11479</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/SwatchColorPicker-mount.html">11545</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/TagPicker-mount_base.html">2664</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/TagPicker-mount.html">2643</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/TeachingBubble-mount_base.html">13466</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/TeachingBubble-mount.html">13442</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Text-mount_base.html">462</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Text-mount.html">476</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/TextField-mount_base.html">1481</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/TextField-mount.html">1452</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ThemeProvider-mount_base.html">1215</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ThemeProvider-mount.html">1244</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ThemeProvider-virtual-rerender_base.html">649</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ThemeProvider-virtual-rerender.html">636</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1950</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1938</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Toggle-mount_base.html">867</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/Toggle-mount.html">848</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/buttonNative-mount_base.html">155</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21576/perf-test/buttonNative-mount.html">153</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9423"/>

**Review by miroslavstastny — APPROVED** [review]

File: packages/react-card/src/components/CardHeader/renderCardHeader.tsx (hunk 2 above)
Comment (khmakoto) [reply packages/react-card/src/components/CardHeader/renderCardHeader.tsx:16]: CCing @andrefcdias

Comment (andrefcdias) [reply packages/react-card/src/components/CardHeader/renderCardHeader.tsx:16]: Yes, but now that you mention it, behavior is weird. Content only exists due to our policy of "every container is a slot", even though it's only there for styling the content as is designed in Fluent.
I'll add this problem to my list of stuff to resolve.

**Review by andrefcdias — APPROVED** [review]

**Comment by miroslavstastny** [comment]
Let's force merge to unblock release.

**Pull request merged**